en/flight_controller/pixhawk3_pro.md

```diff
@@ -64,4 +64,13 @@ For information about wiring and using this port see:
 <!-- Add SWD cross link later -->
 
 
+## Serial Port Mapping
 
+UART | Device | Port
+--- | --- | ---
+UART1 | /dev/ttyS0 | WiFi
```

Comment:
Note, this may need to change in response to https://github.com/PX4/px4_user_guide/pull/672/files#r388712962 - ie is UART1 correct for FMUv4 pro